package.json

```diff
@@ -31,6 +31,7 @@
     "axios": "^0.21.1",
     "cookie": "^0.4.1",
     "date-fns": "^2.16.1",
+    "dayjs": "^1.11.7",
```

Comment:
What method did you use to make this conclusion? On what basis is this better than moment? On the basis that everyone is familiar with moment and non of the devs have used dayJS before?

Reply:
Firstly, according to the[ readme](https://yarnpkg.com/package/moment), `Moment.js` is now a legacy project. They recommend using a different library.

I selected `dayjs` because it was built to use the same API at Moment. So I see no issue for other developers to use it.

Reply:
Good logic